11/debian-9/rootfs/libpostgresql.sh

```diff
@@ -0,0 +1,694 @@
+#!/bin/bash
+#
+# Bitnami PostgreSQL library
+
+# shellcheck disable=SC1090
+# shellcheck disable=SC1091
+
+# Load Generic Libraries
+. /libfile.sh
+. /liblog.sh
+. /libservice.sh
+. /libvalidations.sh
+
+########################
+# Configure libnss_wrapper so PostgreSQL commands work with a random user.
+# Globals:
+#   POSTGRESQL_*
+# Arguments:
+#   None
+# Returns:
+#   None
+#########################
+postgresql_enable_nss_wrapper() {
+  if ! getent passwd "$(id -u)" &> /dev/null && [ -e "$NSS_WRAPPER_LIB" ]; then
+    debug "Configuring libnss_wrapper..."
+    export LD_PRELOAD="$NSS_WRAPPER_LIB"
+    # shellcheck disable=SC2155
+    export NSS_WRAPPER_PASSWD="$(mktemp)"
+    # shellcheck disable=SC2155
+    export NSS_WRAPPER_GROUP="$(mktemp)"
+    echo "postgres:x:$(id -u):$(id -g):PostgreSQL:$POSTGRESQL_DATA_DIR:/bin/false" > "$NSS_WRAPPER_PASSWD"
+    echo "postgres:x:$(id -g):" > "$NSS_WRAPPER_GROUP"
+  fi
+}
+
+########################
+# Load global variables used on PostgreSQL configuration.
+# Globals:
+#   POSTGRESQL_*
+# Arguments:
+#   None
+# Returns:
+#   Series of exports to be used as 'eval' arguments
+#########################
+postgresql_env() {
+    declare_env_alias() {
+      local -r alias="${1:?missing environment variable alias}"
+      local -r original="${2:?missing original environment variable}"
+
+      if env | grep -q "${original}"; then
+          cat << EOF 
+export $alias="${!original}"
+EOF
+      fi
+    }
+  
+    # Alias created for official PostgreSQL image compatibility
```

Comment:
Only three alias? The nami version declared more alias no?

Reply:
There are more below, inside the loop

Reply:
rigth!